src/classes/DatabaseManager.js

```diff
@@ -35,47 +35,48 @@ class DatabaseManager {
    * Insert a guild into the guild_settings table.
    * @param  {String}             id     The ID of the guild.
    * @param  {String}             prefix The default prefix
-   * @return {(Number|undefined)}        Returns 0 on success or undefined.
+   * @return {(Array|undefined)}        Returns 0 on success or undefined.
    */
   async addClient (id, prefix) {
-    return this._qb.run('insert', { table: 'guild_settings', data: { id, prefix } })
-      .then(() => this._qb.run('insert', { table: 'guild_toggles', data: { id } }))
+    return this._qb.insert({ table: 'guild_settings', data: { id, prefix } })
+      .then(() => this._qb.insert({ table: 'guild_toggles', data: { id } }))
   }
 
   /**
    * Insert a status into the statuses table.
    * @param  {String}             name   The name of the status.
    * @param  {Number}             [type] The type of the status.
+   * @param  {Boolean}            [def]  Whether or not this is the default status.
    * @return {(Number|undefined)}        Returns 0 on success or undefined.
```

Comment:
This is just one of MANY examples, but Simple-Knex was updated to return the updated database after an alteration, rather than a success number.